apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -55,32 +56,22 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
 
         raw: RAW_DATA_FIELD,
 
-        /**
-         * A note on toPay:
-         *
-         * We store money in all currencies as a string, with (.) as delimiter:
-         * We store currency in BillingIntegration in this format. This information helps client to distinguish the partial part:
-         *
-         * refs: https://gist.github.com/Fluidbyte/2973986
-         *
-         * "RUB": {
-         *		"symbol": "RUB",
-         *		"name": "Russian Ruble",
-         *		"symbol_native": "₽.",
-         *		"decimal_digits": 2,
-         * 		"rounding": 0,
-         *		"code": "RUB",
-         *		"name_plural": "Russian rubles"
-         *	},
-         *
-         * 123.30 (for 123 Ruble and 30 Kopeck)
-         * 6.20 (for 6$ and 20 Cents)
-         *
-         */
         toPay: {
             schemaDoc: 'Total sum to pay. Usually counts as the sum of all services. Stored like this: "123.30" Detail level 1.',
-            type: Text,
+            type: Decimal,
             isRequired: true,
+            knexOptions: {
+                scale: 2,
+            },
+        },
+
+        commission: {
```

Comment:
why is commission here?

Reply:
As for me, the commissions are related to payment rather than receipts. It's a good question